src/app/purchase/order-form/order-form.component.ts

```diff
@@ -254,10 +256,11 @@ export class OrderFormComponent implements OnInit {
     let token = sessionStorage.getItem('token');
     let decoded = this.jwtHelper.decodeToken(token);
 
-    this.vatRate = decoded.PC_VAT ? decoded.PC_VAT : 7;
-    this.currentVatRate = decoded.PC_VAT ? decoded.PC_VAT : 7;
+    this.vatRateTmp = decoded.PC_VAT
+    this.vatRate = this.vatRateTmp
+    this.currentVatRate = decoded.PC_VAT
```

Comment:
ในกรณีที่ไม่มีการกำหนดค่า `PC_VAT` บรรทัดนี้จะ error นะครับ แนะนำให้ใส่แบบเดิม `this.currentVatRate = decoded.PC_VAT ? decoded.PC_VAT : 7;`